tensorflow/python/ops/nn_grad.py

```diff
@@ -210,36 +210,36 @@ def _BiasAddGradGrad(op, received_grad):
   Args:
     op: BiasAddGrad op for which we are calculating gradients.
     received_grad: The gradients passed to the BiasAddGrad op.
-    
+
   Returns:
     A single gradient Tensor for the input to BiasAddGrad (which
     is the gradient of the bias term in BiasAdd)
   """
-  
+
   try:
     data_format = op.get_attr("data_format")
   except ValueError:
     data_format = None
-  
+
   shape = array_ops.shape(op.inputs[0])
   rank = array_ops.rank(op.inputs[0])
   bias_shape = array_ops.shape(received_grad)
-  
+
   if data_format == b"NCHW":
     expanded_shape = array_ops.concat(
       0,
       [array_ops.ones_like(shape[:-3]), bias_shape, array_ops.ones_like(shape[-2:])]
     )
-    
+
     tile_mults = array_ops.concat(0, [shape[:-3], [1], shape[-2:]])
-    
+
```

Comment:
@thuyen Just realized these two empty lines may not be needed. Could you take a look?